static/app/views/usageStats/usageChart/index.tsx

```diff
@@ -199,17 +205,23 @@ export class UsageChart extends React.Component<Props, State> {
   }
 
   get chartColors() {
-    const {dataCategory} = this.props;
+    const {dataCategory, chartHighlightAccepted} = this.props;
 
     if (dataCategory === DataCategory.ERRORS) {
-      return [COLOR_ERRORS, COLOR_ERRORS_DROPPED, COLOR_PROJECTED];
+      return chartHighlightAccepted
+        ? [COLOR_ERRORS, COLOR_ERRORS_LIGHT, COLOR_PROJECTED]
+        : [COLOR_ERRORS_LIGHT, COLOR_ERRORS, COLOR_PROJECTED];
```

Comment:
is this prop needed? I think it'd be nice for all usage charts to use the same colors per outcome

Reply:
For the subscription page, it's better to have an emphasis on `accepted`, so it's easy to see usage against quota.

For the org stats page, product wants the page to aid sales folks, so highlighting `dropped` is more useful.